README.md

```diff
@@ -869,124 +871,210 @@ After creating an initialization request successfully you can check at which sta
 }
 ```
 
-## ZOOP INCOME TAX RETURN (ITR) GATEWAY (Beta)
+## ZOOP INCOME TAX DEPARTMENT (ITD) GATEWAY (Beta)
 
-<a name="itrIntro"></a>
+<a name="itdIntro"></a>
 
 ### 1. Introduction
 
-Income Tax Return is the form in which assessee files information about his Income and tax thereon to Income Tax Department. The Zoop ITR Gateway allows allows you to fetch the tax returns of your client and make better business decision on them.
+Income Tax Department is the form in which assessee files information about his Income and tax thereon to Income Tax Department. The Zoop ITD Gateway allows allows you to fetch the tax returns of your client and make better business decision.
 
```

Comment:
there are two allows at after Gateway.

Reply:
Income Tax Department holds and verifies the earnings and tax related information of an individual. 
Income tax return is a form where a person submits information about his income earned and the tax paid to the Department. Whereas 26AS is a consolidated annual tax credit statement of an individual's earnings as per the ITD records. The Zoop ITD Gateway allows you to fetch and verify the financial information of your client in order to make better business decisions.